internal/exec/engine.go

```diff
@@ -103,6 +110,41 @@ func (e Engine) Run(stageName string) error {
 	return nil
 }
 
+// isConfigCacheExist checks if we already have a cached config.
+// TODO(sohan): This is a hack to emit the journald message only
+// once for the base config (https://github.com/coreos/ignition/issues/950)
+func (e *Engine) isConfigCacheExist(cache string) error {
+	var haveConfig bool
+	if _, err := os.Stat(e.ConfigCache); err != nil {
+		if !os.IsNotExist(err) {
+			return err
+		}
+		haveConfig = true
```

Comment:
That should be `false`, no? And then in the `else` it'd be `true`.

Reply:
Fixed.